app/src/features/surveys/animals/profile/captures/capture-form/edit/EditCapturePage.tsx

```diff
@@ -143,10 +132,12 @@ export const EditCapturePage = () => {
         captures: [
           {
             capture_id: values.capture.capture_id,
-            capture_timestamp: captureTimestamp,
-            release_timestamp: releaseTimestamp,
-            capture_comment: values.capture.capture_comment ?? '',
-            release_comment: values.capture.release_comment ?? '',
+            capture_date: values.capture.capture_date,
+            capture_time: values.capture.capture_time || undefined,
+            release_date: values.capture.release_date || undefined,
+            release_time: values.capture.release_time || undefined,
```

Comment:
Does the release date/time need to default to the capture date/time if not provided, like it does in the CreateCapturePage.tsx?

Reply:
Nice catch. Fixing...